saleor/webhook/serializers.py

```diff
@@ -28,3 +34,38 @@ def serialize_checkout_lines(checkout: "Checkout") -> List[dict]:
             }
         )
     return data
+
+
+def serialize_product_attributes(
+    product: Union["Product", "ProductVariant"]
+) -> List[dict]:
+    queryset = (
+        (
+            AssignedProductAttribute.objects.filter(product_id=product.id)
+            if isinstance(product, Product)
+            else AssignedVariantAttribute.objects.filter(variant_id=product.id)
```

Comment:
Shouldn't be:
```suggestion
            else AssignedVariantAttribute.objects.filter(variant__product_id=product.id)
```

Reply:
No :) It's good. It's a Product or productVariant

Reply:
True, sorry :)